.github/workflows/inductor-perf-test-nightly-x86.yml

```diff
@@ -1,6 +1,9 @@
 name: inductor-perf-nightly-x86
 
 on:
+  pull_request:
+    paths:
+      - .github/workflows/inductor-perf-test-nightly-x86.yml
```

Comment:
Nit: not really necessary since the file will be updated occasionally, and you can always manually trigger a run when needed.